packages/docs/docs/lambda/serverlesslambdasetup.md

```diff
@@ -0,0 +1,203 @@
+---
+image: /generated/articles-docs-lambda-serverlesslambdasetup.png
+id: serverlesslambdasetup
+title: Serverless lambda setup
+slug: /lambda/serverless-lambdas-setup
+crumb: "Lambda"
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This is a follow up write up for a serverless function from [lamdbda permissions](/docs/lambda/lambda-role-permissions).
+
+### Prequisites
+- Make sure that you're local AWS profile is able to deploy to AWS.
+
+### Setup
+
+#### 1. Clone or download the project 
+Project is located inside the [`remotion project`](https://github.com/alexfernandez803/example-lambda) 
```

Comment:
Let's call it "example project" or "reference project"

Reply:
completed.